spec/models/restaurant_spec.rb

```diff
@@ -3,15 +3,30 @@
 RSpec.describe Restaurant, type: :model do
   describe 'DB Table' do
     it { is_expected.to have_db_column :name }
+    it { is_expected.to have_db_column :city }
+    it { is_expected.to have_db_column :post_code }
+    it { is_expected.to have_db_column :street_address }
+    it { is_expected.to have_db_column :longitude }
+    it { is_expected.to have_db_column :latitude }
   end
 
   describe 'Validations' do
     it { is_expected.to validate_presence_of :name }
+    it { is_expected.to validate_presence_of :city }
+    it { is_expected.to validate_presence_of :post_code }
+    it { is_expected.to validate_presence_of :street_address }
   end
 
   describe 'Factory' do
     it 'should be valid' do
       expect(FactoryBot.create(:restaurant)).to be_valid
     end
   end
+
+  before do
+    FactoryBot.create(:restaurant)
+  end
+  it 'responds to full_address' do
+    expect(Restaurant.first).to respond_to(:full_address)
```

Comment:
here @tochman

Reply:
Okay. That is a very basic test. Thank you

Reply:
If you are satisfied with this standard of testing rhen I will not interfere.

Reply:
Any suggestions on how to improve the test?

Reply:
Yes, take a look at betterspecs.org. 

You know, use `describe`, follow a standard (like referencing method names tith `#`), check for the actual values of `full_address`, etc...)